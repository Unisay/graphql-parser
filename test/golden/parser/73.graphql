enum Direction {
  NORTH
  EAST
  SOUTH
  """
  West
  """
  WEST
}
