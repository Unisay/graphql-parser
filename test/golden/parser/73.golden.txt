Right
    ( Document
        ( DefinitionTypeSystem
            ( TypeSystemDefinitionType
                ( TypeDefinitionEnum
                    ( EnumTypeDefinition
                        { _etdDescription = Nothing
                        , _etdName = Name "Direction"
                        , _etdDirectives = []
                        , _etdValuesDefinition = Just
                            ( EnumValuesDefinition
                                [ EnumValueDefinition
                                    { _evdDescription = Nothing
                                    , _evdEnumValue = Name "NORTH"
                                    , _evdDirectives = []
                                    }
                                , EnumValueDefinition
                                    { _evdDescription = Nothing
                                    , _evdEnumValue = Name "EAST"
                                    , _evdDirectives = []
                                    }
                                , EnumValueDefinition
                                    { _evdDescription = Nothing
                                    , _evdEnumValue = Name "SOUTH"
                                    , _evdDirectives = []
                                    }
                                , EnumValueDefinition
                                    { _evdDescription = Just
                                        ( Description "West" )
                                    , _evdEnumValue = Name "WEST"
                                    , _evdDirectives = []
                                    }
                                ]
                            )
                        }
                    )
                )
            ) :| []
        )
    )